recipes/zpp_throwing/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,18 @@
+from conans import ConanFile, CMake
+from conan.tools.build import cross_building
```

Comment:
```suggestion
from conans import ConanFile, CMake, tools
```
Please, use the legacy tools for cross_building when testing Conan v1, then we can prove that's compatible :)

Reply:
@uilianries 
Thank you for your comment.
I understand.